core/src/main/java/hudson/model/listeners/ItemListener.java

```diff
@@ -212,28 +197,16 @@ public static void checkBeforeCopy(final Item src, final ItemGroup parent) throw
     }
 
     public static void fireOnCreated(final Item item) {
-        forAll(l -> {
-            if (l != null) {
-                l.onCreated(item);
-            }
-        });
+        Listeners.notify(ItemListener.class, l -> l.onCreated(item));
     }
```

Comment:
With this change, the following tests start failing:

- `com.cloudbees.hudson.plugins.folder.properties.AuthorizationMatrixPropertyTest#ensureCreatorHasPermissions`
- `hudson.security.ProjectMatrixAuthorizationStrategyTest#ensureCreatorHasPermissions`
- `hudson.security.ProjectMatrixAuthorizationStrategyTest#ensureNoJobPropertyDuplication`
- `org.jenkinsci.plugins.matrixauth.AuthorizationMatrixNodePropertyTest#ensureCreatorHasPermissions`

Without this change, the tests still pass.